sdk/iot/Azure.Iot.Hub.Service/src/API Design.md

```diff
@@ -1,24 +1,86 @@
-﻿
+﻿# Azure Iot Hub Service API Design Doc
+This document outlines the APIs for the Azure Iot Hub Service SDK
 
+<details><summary><b>Constructors</b></summary>
+    
+```csharp
 
+```
+</details>
 
-# Azure Iot Hub Service API Design Doc
-This document outlines the APIs for the Azure Iot Hub Service SDK
+<details><summary><b>Configuration</b></summary>
+    
+```csharp
+
+```
+</details>
+
+<details><summary><b>Statistics</b></summary>
+
+```csharp
+
+```
+</details>
+
+<details><summary><b>Registry</b></summary>
```

Comment:
A note here, stating that these relate to IoT Hub identity registry operation, would give some additional context.